pkg/util/slice/slice.go

```diff
@@ -48,3 +48,14 @@ func ShuffleStrings(s []string) []string {
 	}
 	return shuffled
 }
+
+// Int64Slice attaches the methods of Interface to []int64,
+// sorting in increasing order.
+type Int64Slice []int64
+
+func (p Int64Slice) Len() int           { return len(p) }
+func (p Int64Slice) Less(i, j int) bool { return p[i] < p[j] }
+func (p Int64Slice) Swap(i, j int)      { p[i], p[j] = p[j], p[i] }
+
+// Sorts []int64 in increasing order
+func Ints64(a []int64) { sort.Sort(Int64Slice(a)) }
```

Comment:
unless i am missing something, this function should be called sort instead of Ints64